src/devices/cpu/m6800/m6800.h

```diff
@@ -308,6 +308,8 @@ class m6800_cpu_device :  public cpu_device
 	void eorb_im();
 	void eorb_ix();
 	void illegal();
+	void illega2();
+	void illega3();
```

Comment:
Could you drop a vowel rather than a consonant, and also make the illegal 1-byte opcode handler use the same format for consistency?  It's easier to read.  Maybe call them `illegl1`, `illegl2` and `illegl3`.